common/religion/doctrines/00_core_tenets.txt

```diff
@@ -350,72 +384,79 @@
 		}
 	}
 
-	tenet_gnosticism = {
-		icon = core_tenet_gnostic
-		piety_cost = {
-			if = {
-				limit = {
-					OR = {
-						religion_tag = dualism_religion
-						religion_tag = zoroastrianism_religion
-					}
-				}
-				value = faith_tenet_cost_low
-			}
-			else_if = {
-				limit = {
-					OR = {
-						religion_tag = judaism_religion
-						religion_tag = islam_religion
-						religion_tag = hinduism_religion
-						has_doctrine = pagan_hostility_doctrine
-					}
-				}
-				value = faith_tenet_cost_high
-			}
-			else = {
-				value = faith_tenet_cost_mid
-			}
-
-			# Multiplier for keeping same tenet
-			if = {
-				limit = { has_doctrine = tenet_gnosticism }
-				multiply = faith_unchanged_doctrine_cost_mult
-			}
-		}
-
-		traits = {
-			virtues = { temperate }
-			sins = { gluttonous }
-		}
-
-		can_pick = {
-			custom_description = {
-				text = incompatible_tenet_eastern_syncretism_trigger
-				NOT = {
-					flag:tenet_eastern_syncretism = { is_in_list = selected_doctrines }
-				}
-			}
-		}
-
-		parameters = {
-			hostility_override_tenet_gnosticism = 0
-			granting_titles_gives_stress = yes
-		}
-
-		character_modifier = {
-			learning = 2
-			stewardship = -2
-		}
-	}
+	# Warcraft
+	# tenet_gnosticism = {
+		# icon = core_tenet_gnostic
+		# piety_cost = {
+			# if = {
+				# limit = {
+					# OR = {
+						# religion_tag = dualism_religion
+						# religion_tag = zoroastrianism_religion
+					# }
+				# }
+				# value = faith_tenet_cost_low
+			# }
+			# else_if = {
+				# limit = {
+					# OR = {
+						# religion_tag = judaism_religion
+						# religion_tag = islam_religion
+						# religion_tag = hinduism_religion
+						# has_doctrine = pagan_hostility_doctrine
+					# }
+				# }
+				# value = faith_tenet_cost_high
+			# }
+			# else = {
+				# value = faith_tenet_cost_mid
+			# }
+
+			# # Multiplier for keeping same tenet
+			# if = {
+				# limit = { has_doctrine = tenet_gnosticism }
+				# multiply = faith_unchanged_doctrine_cost_mult
+			# }
+		# }
+
+		# traits = {
+			# virtues = { temperate }
+			# sins = { gluttonous }
+		# }
+
+		# can_pick = {
+			# custom_description = {
+				# text = incompatible_tenet_eastern_syncretism_trigger
+				# NOT = {
+					# flag:tenet_eastern_syncretism = { is_in_list = selected_doctrines }
+				# }
+			# }
+		# }
+
+		# parameters = {
+			# hostility_override_tenet_gnosticism = 0
+			# granting_titles_gives_stress = yes
+		# }
+
+		# character_modifier = {
+			# learning = 2
+			# stewardship = -2
+		# }
+	# }
```

Comment:
I suggest keeping it simple and just delete it if we are not going to use it, leaving a comment about what it was
```suggestion
	# Warcraft
	# tenet_gnosticism
```